### PR TITLE
Adds tails to fox infusion

### PR DESCRIPTION
**Pull request by spiders24110**
## About The Pull Request
Fox infusion now gives fox tail, and also fixed fox ears being invisible.
Amazing sprites by @Imaginos16

https://github.com/user-attachments/assets/6cba6cac-8f71-4021-8216-4d05e1f97863
## Why It's Good For The Game
We already have the infusion, now it's complete. Though maybe gonna add additional tail variants later.
## Changelog
:cl: Imaginos, Striders13
add: fox infusion now gives fox tail
/:cl:

### CODE REVIEW

**Comment by zoomachina** [comment]
the gates of hell have opened

File: code/datums/dna.dm
Comment (Ghommie) [inline code/datums/dna.dm:251]: Unrelated note, but dna feature blocks should be refactored at some point. Maybe I'll have time for it this weekend.

**Review by Ghommie — COMMENTED** [review]
No glaring issue codewise at a quick glance. This is more or less how I've done fish tails (which I'm going to improve soon).

**Comment by Timberpoes** [comment]
THIS WILL SAVE SYBIL

**Review by Timberpoes — APPROVED** [review]
THIS WILL SAVE TGSTATION IT MUST BE MERGED.

**Comment by Iamgoofball** [comment]
this is currently hard blocked by @optimumtact so you'll need headcoder approval

**Comment by Striders13** [comment]
> stuff

thats not an animal race tho, that's just adding a tail to already existing infusion

also
изображение

**Review by Mothblocks — CHANGES_REQUESTED** [review]
image

Only fix the ears plz

**Comment by DrAmazing343** [comment]
It's mothblover foxbros...

**Comment by scriptis** [comment]
> no stupid fox shit

this isnt stupid its soul whats the problem

**Comment by castawaynont** [comment]
mothblocks is responsible for the decline of fox population in north america

**Comment by Xkallubar** [comment]
abhorrent

**Comment by MercuryArrow** [comment]
I don't know why we're even keeping the infusion honestly if every attempt to at least try to finish it gets shut down.
May as well drop the fly and moth infusions too, or does the logic only apply to "cute, furry" animals.

**Comment by BHat0451** [comment]
> I don't know why we're even keeping the infusion honestly if every attempt to at least try to finish it gets shut down. May as well drop the fly and moth infusions too, or does the logic only apply to "cute, furry" animals.

This is extremely true. Fox ears have existed literally since infusion became a thing and it would only make sense to add the only other component to the infusion, or otherwise remove the infusion entirely.

**Comment by Blacklist897** [comment]
either remove it or finish it cowards

**Comment by DrAmazing343** [comment]
give me swishy swish or give me death

**Comment by LemonInTheDark** [comment]
No tails

**Comment by carlarctg** [comment]
we should fork over this

**Comment by Iamgoofball** [comment]
@MrStonedOne you'll need to step in here, 2 headmins want it and 2 headcoders have veto'd it so we're at a standstill

**Comment by MrStonedOne** [comment]
@dendyd00m break the tie since burger wont. (this is non-binding) (in fact i expect the headcoders to ignore this)

**Pull request closed without merging**
